portality/article.py

```diff
@@ -156,10 +156,12 @@ def discover_duplicates(article, owner=None):
         if len(dois) > 0:
             # there should only be the one
             doi = dois[0]
-            articles = models.Article.duplicates(issns=issns, doi=doi)
-            possible_articles['doi'] = [a for a in articles if a.id != article.id]
-            if possible_articles['doi']:
-                found = True
+            # Only perform de-duplication on genuine DOIs - todo: do we need a full regex check?
```

Comment:
Probably not - DOIs are relatively free-form after the prefix.  I think you have to have one or more digits following the ".", but after that it's up to you what you do with it.  So, starting with 10. is sufficient, I'd say.